submissions/pozzitive11/popup/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hooli-style Popup</title>
+  <link rel="icon" type="image/x-icon" href="/images/favicon.png">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header">
+    <div class="container">
+      <nav class="nav" title="Main navigation">
+        <ul class="list-reset nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Mail
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Images
+            </a>
+          </li>
+          <li class="nav__item nav__item-btn">
+            <input id="popup" class="popup__input" type="checkbox">
+            <label class="nav__btn popup__btn" for="popup">
+              <img class="popup__img" src="./images/popup-button.png" alt="popup">
+            </label>
+            <div class="popup__menu">
+              <div class="menu__main">
```

Comment:
Logically it looks like a list, nested list makes it easier to understand navigation structure.